.github/workflows/lint.yml

```
@@ -75,6 +75,9 @@ jobs:
       - name: Running mypy on vsmasktools
         run: uv run mypy vsscale
 
+      - name: Running mypy on vsmasktools
```

copypasta (and more above but don't have to fix those here)
```suggestion
      - name: Running mypy on vsaa
```